docs/content/_apidocs.mdx

```diff
@@ -88,6 +88,10 @@ Dagster also provides a growing set of optional add-on libraries to integrate wi
 
 - [GCP](/\_apidocs/libraries/dagster-gcp) (`dagster-gcp`) Dagster integrations for working with Google Cloud Platform resources.
 
+- [GCP+Pandas](/\_apidocs/libraries/dagster-gcp-pandas) (`dagster-gcp-pandas`) Dagster integrations for working with Google Cloud Platform resources with Pandas DataFrames.
+
+- [GCP+PySpark](/\_apidocs/libraries/dagster-gcp-pyspark) (`dagster-gcp-pyspark`) Dagster integrations for working with Google Cloud Platform resources with PySpark DataFrames.
```

Comment:
On the API reference page itself, one of the first headings says `BigQuery` but the description here doesn't mention that at all. Does this work with all GCP resources or just BigQuery?

Reply:
right now there's only bigquery support, but in the future this library could have things for other GCP resources

Reply:
:+1: D'you think it's worth mentioning in the description here? I know some folks have asked about support for BQ specifically, so my thinking was that would make it easier to find if they're skimming